### PR TITLE
Improve PR description of the automatic PR

### PR DESCRIPTION
**Pull request by wtrocki**
## Description

Add note about the need to reopen PR to trigger builds.

### CODE REVIEW

File: .github/workflows/autoupdate-prod.yaml [hunk 1]
```diff
@@ -53,23 +53,25 @@ jobs:
...
+            1. If the build is failing please check the logs and fix the issues in the OpenAPI file (openapi/atlas-api.yaml)
+            2. Do not attempt to manually fix any issues in the Golang code as all changes are automatically generated. 
+            Fixes should be applied to the OpenAPI file (openapi/atlas-api.yaml) or ./tools engine in a separate PR.
 
             ## Manual Review Procedure
 
             1. Review changes in the OpenAPI file (openapi/atlas-api.yaml)
-            2. If we see any breaking changes without new resource version (
-              - Review breaking changes listed in `Detect Breaking Changes` action
+            2. If we see any breaking changes without a new resource version.
```
Comment (andreaangiolillo) [inline .github/workflows/autoupdate-prod.yaml:67]:
```suggestion
            2. If we see any breaking changes without a new resource version.
```

What does "without a new version" mean here? Could you provide an example?

Comment (wtrocki) [reply .github/workflows/autoupdate-prod.yaml:67]: https://github.com/mongodb/atlas-sdk-go/blob/main/docs/doc_1_concepts.md#example-version-v2023020100100

Comment (wtrocki) [reply .github/workflows/autoupdate-prod.yaml:67]:
```suggestion
            2. If we see any breaking changes [without a new resource version](https://github.com/mongodb/atlas-sdk-go/blob/main/docs/doc_1_concepts.md#example-version-v2023020100100) .
```

Comment (andreaangiolillo) [reply .github/workflows/autoupdate-prod.yaml]: mh.. sorry but it is still not clear 😭 I am wondering if we should look for a specific property in the code or something else. What you shared is only the versioning strategy of the SDK.

Comment (andreaangiolillo) [reply .github/workflows/autoupdate-prod.yaml]: Thanks for the clarification!

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Pull request merged**